ciri/api/tests/test_get_trytes.c

```diff
@@ -46,11 +46,11 @@ void test_get_trytes_not_found(void) {
   get_trytes_req_t *req = get_trytes_req_new();
   get_trytes_res_t *res = get_trytes_res_new();
   tryte_t tx_trytes[NUM_TRYTES_SERIALIZED_TRANSACTION];
+  flex_trit_t null_hash[FLEX_TRIT_SIZE_243] = {FLEX_TRIT_NULL_VALUE};
```

Comment:
This is not initializing all the values to `FLEX_TRIT_NULL_VALUE`, only the first one then zeros all the others.

Reply:
oops, thanks.